### PR TITLE
LPS-151540 Create an Util to provide predefined variables to scripts

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:relevant

File: modules/apps/object/object-api/src/main/java/com/liferay/object/util/ObjectScriptVariablesUtil.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+			String contentType = _getContentType(
+				dtoConverterRegistry, objectDefinition,
+				systemObjectDefinitionMetadataTracker);
+
+			allVariables.putAll(
+				(Map<String, Object>)payloadJSONObject.get(
+					"model" + objectDefinition.getName()));
+			allVariables.putAll(
+				(Map<String, Object>)payloadJSONObject.get(
+					"modelDTO" + contentType));
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-api/src/main/java/com/liferay/object/util/ObjectScriptVariablesUtil.java:59]: Can we `_getContentType` inline?

File: modules/apps/object/object-api/src/main/java/com/liferay/object/util/ObjectScriptVariablesUtil.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+		}
+		else {
+			allVariables.putAll(
+				(Map<String, Object>)payloadJSONObject.get("objectEntry"));
+
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-api/src/main/java/com/liferay/object/util/ObjectScriptVariablesUtil.java:64]: Delete this blank line

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-api/src/main/java/com/liferay/object/util/ObjectScriptVariablesUtil.java (hunk 1 above)
Comment (seltong) [reply modules/apps/object/object-api/src/main/java/com/liferay/object/util/ObjectScriptVariablesUtil.java:59]: Hi @MarinhoFeliphe, 
I tried doing this but SF recommended creating a variable just like I did

**Comment by seltong** [comment]
ci:help

**Comment by seltong** [comment]
ci:test:stable

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:stable

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:stable

**Comment by seltong** [comment]
Resent here: https://github.com/brianchandotcom/liferay-portal/pull/124083

**Comment by anabuchmann** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:reevaluate:1307401_4879

**Comment by seltong** [comment]
ci:forward

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:forward

**Comment by seltong** [comment]
ci:forward

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:stop

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
Resent here: https://github.com/brianchandotcom/liferay-portal/pull/124727

**Pull request closed without merging**